enterprise/ha/src/docs/dev/arbiter.asciidoc

```diff
@@ -9,8 +9,8 @@ Instead, what is called arbiter instances can be deployed.
 They can be regarded as cluster participants in that their role is to take part in master elections with the single purpose of breaking ties in the election process.
 That makes possible a scenario where you have a cluster of 2 Neo4j database instances and an additional arbiter instance and still enjoy tolerance of a single failure of either of the 3 instances.
 
-Arbiter instances are configured in the same way as Neo4j HA members are -- through the _conf/neo4j.conf_ file.
-The instance is configured to be an arbiter by setting the +org.neo4j.server.database.mode+ option in _conf/neo4j.conf_ to +ARBITER+.
+Arbiter instances are configured in _neo4j.conf_ using the same settings as standard Neo4j HA members.
+The instance is configured to be an arbiter by setting the +org.neo4j.server.database.mode+ option in _neo4j.conf_ to +ARBITER+.
```

Comment:
Second reference to the config file is unnecessary, `in _neo4j.conf_` can be removed. Please substitute backtick for plus around the setting name and value in this sentence.